### DISCUSSION TITLE
Reading gyro

### DISCUSSION THREAD

**Discussion by WhittyRobot1** [pybricks/support#1852] [Q&A]
I don't know how to see gyroscope readings  in pybricks  can someone help me?

**Comment by BertLindeman** [comment]
A short answer from me to get you a bit on the way.

It depends on what you want to know / use.

Most data is in the `input` blocks.

Like there: 
![image](https://github.com/user-attachments/assets/f9c31d19-dbc8-49cd-8ebb-55546b4c2aff)

Bert

**Comment by BertLindeman** [comment]
Did this help you?

If not how can I help you?

Bert
